lms/static/js/discussions_management/models/course_discussions_settings.js

```diff
@@ -0,0 +1,14 @@
+(function(define) {
```

Comment:
The models/divide_discussions.js file is a model used to contain the detailed information of all the available course wide and inline discussions.

The models/course_discussions_settings.js model is used to retrieve the currently selected discussions to be divided.  It will contain a list of discussion topics that are selected to be divided. 

I have renamed divide_discussions to course_discussions_detail.js for clarity.